deploy_heroku.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import json
+import os
+import subprocess
+import urllib.request
+from typing import List
+
+
+def get_latest_heroku_release(app: str, api_key: str):
+    request = urllib.request.Request(f"https://api.heroku.com/apps/{app}/releases")
+    for key, value in (
+            ("Accept", "application/vnd.heroku+json; version=3"),
+            ("Authorization", f"Bearer {api_key}"),
+            ("Content-Type", "application/json"),
+            ("Range", "version ..; order=desc,max=10;"),
+    ):
+        request.add_header(key, value)
+
+    with urllib.request.urlopen(request) as response:
+        assert response.getcode() == 200, response.getcode()
```

Comment:
Das ist sehr interessant - so habe ich angefangen, aber dann als ich es live (denke mal mit sepacetamol) ausprobiert habe kam in der tat 206, also ich habe ich geändert - kann es sein, dass wenn du weniger als 10 hast kommt 206? Dann kann man die Prüfung machen nach dem motto 

```
assert 200 <= response.getcode() < 300, response.getcode()
```

Reply:
Das werde ich mal testen